### PR TITLE
[FOLIO-41] sent email with token

### PR DESCRIPTION
**Pull request by RashmiDholakiya**
**What's this PR do?**
It will send email with token 

**Where should the reviewer start?**
from userController, Email will be sent to new user.

**What are the relevant tickets?**
https://arsenaltech.atlassian.net/browse/FOLIO-41

### CODE REVIEW

File: app/Http/Requests/UserRequest.php [hunk 1]
```diff
@@ -19,9 +21,15 @@ public function authorize()
 	 */
 	public function rules()
 	{
+
+		if (Input::get('user_id') != NULL) {
```
Comment (poojajajal) [inline app/Http/Requests/UserRequest.php]: This is wrong, why we need to write separate rule. We can manage it by one.

File: app/Http/Controllers/Auth/PasswordController.php [hunk 2]
```diff
@@ -22,18 +22,24 @@ class PasswordController extends Controller
 	| explore this trait and override any methods you wish to tweak.
 	|
 	*/
-	public function getResetPassword($email)
+	public function getResetPassword($email, $token)
```
Comment (poojajajal) [inline app/Http/Controllers/Auth/PasswordController.php]: This function for use reset password. Use other function with appropriate place.

File: app/Http/Requests/UserRequest.php (hunk 1 above)
Comment (RashmiDholakiya) [inline app/Http/Requests/UserRequest.php]: Done

File: app/Http/Controllers/Auth/PasswordController.php (hunk 2 above)
Comment (RashmiDholakiya) [inline app/Http/Controllers/Auth/PasswordController.php]: Done

**Comment by poojajajal** [comment]
Code look good to me . @RashmiDholakiya  You can merge this.

**Pull request merged**